include/mlir/Dialect/Linalg/Utils/Utils.h

```diff
@@ -134,23 +134,27 @@ struct TiledLinalgOp {
 };
 
 /// Performs standalone tiling of a single LinalgOp by `tileSizes`.
-/// Returns a struct containing the tiled loops and the cloned op if successful,
-/// llvm::None otherwise.
+/// and permute the loop nest according to `permutation`
```

Comment:
Can you plz explain what the permutation is and give an example of valid / invalid permutation?
Here and in other places that extend the API.

Reply:
Added explanations with examples and include a validation function